src/features/classification/security-controls/SecurityControlsItem.js

```diff
@@ -2,18 +2,34 @@
 import * as React from 'react';
 import { FormattedMessage, type MessageDescriptor } from 'react-intl';
 
+import { bdlBoxBlue } from '../../../styles/variables';
+import Tooltip from '../../../components/tooltip';
+import IconInfo from '../../../icons/general/IconInfo';
 import './SecurityControlsItem.scss';
 
 type Props = {
+    appNames: ?MessageDescriptor,
```

Comment:
Sorry I think this might be related to some of my other suggestions, but since we switched the approach to returning a `MessageItem` with a `tooltipMessage`, I think it might make more sense for this to be `tooltipMessage` as well. I think `Props` could probably be just:

```
type Props = `MessageItem`;
```